olx.py

```diff
@@ -26,21 +26,52 @@ def flatten(container):
         else:
             yield i
 
-# TODO: remove polish chars
-def city_name(input):
+
+def city_name(city):
+    POLISH_CHARACTERS = {"ą": "a", "ć": "c", "ę": "e", "ł": "l", "ń": "n", "ó": "o", "ś": "s", "ż": "z", "ź": "z"}
```

Comment:
You could do `unicodedata.normalize('NFKD', city.lower()).encode('ascii', 'ignore').decode("utf-8")` - it gives you a lowercase string stripped of diacritics